demos/orderly/src/many-downloads/many-downloads.R

```diff
@@ -0,0 +1,5 @@
+for (i in 1:200) {
+  file <- paste0("A-really-stupidly-lengthily-named-file-", i, ".rds")
+  saveRDS(list(a = 1, b = 2, c = 3), file)
+  orderly::orderly_artefact(description=paste("File", i), file)
+}
```

Comment:
The mrc-6789-repro branch at https://github.com/mrc-ide/packit/pull/252/ said not to merge it, but you're including the changes here anyway. That seems OK to me because it's helpful to have the bug repro case on main, but I'm not sure why https://github.com/mrc-ide/packit/pull/252/ said not to merge it.

Reply:
Oh yeah, sorry I just meant because the changes are in this PR too so there's not really any point in merging the other PR, which I intended to just close.